.github/workflows/test_prs.yml

```diff
@@ -6,7 +6,7 @@ on:
       - develop
 
 jobs:
-  build:
+  build-and-test:
```

Comment:
The step name doesn't make it clear that we were both building and testing.

<img width="844" alt="Screenshot 2024-06-23 at 12 20 30 AM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/863b7c6b-10ec-4ed5-a65e-8b90523d4cb8">